BAH-645| fix bean definitions and dependency injections

krishnanspace opened the pull request:
Fixed the bean annotations according to https://bahmni.atlassian.net/browse/BAH-645

krishnanspace committed:
BAH-645| fix bean definitions and dependency injections

krishnanspace committed:
BAH-645| fix bean definitions and dependency injections

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/Bahmni/openmrs-module-bahmni.ie.apps?pullRequest=1) <br/>All committers have signed the CLA.

binduak committed:
Bindu | BAH-758 | Upgrade WebServices REST Module to 2.24.0

binduak committed:
Vinay, Bindu | BAH-872 | adding dummy class to fix IT tests (#3)

* Siva, Vinay | Fix build failure due to NoClassDefFoundError for class 'CodedOrFreeText'

This error is obtained due to openMRSWebServicesVersion upgrade.
Corresponding commit
https://github.com/Bahmni/openmrs-module-bahmni.ie.apps/commit/86a89adf1f2979cd0ea1388ed42c9aabb20b13fb

Please find the link for reference related to error
https://talk.openmrs.org/t/java-lang-noclassdeffounderror-org-openmrs-allergen-on-using-webservices-rest-2-14-with-openmrs-1-x/6361/3

* Bindu | Moving the dummy class from src folder to test folder

AlekhyaYalla committed:
#BAH-866 | Translations to be copied from one version to next version (#4)

* Alekhya, Vinay | MOBN-407 | Persist form translation on import and update to next version

* Siva, Alekhya | MOBN-644 | Copy translations from imported forms

* Alekhya | MOBN-644 | Change the constant & method names accordingly using ref version

swethab184 committed:
Merge master
